packages/grafana-data/src/types/query.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Attached to query results (not persisted)
+ *
+ * @public
+ */
+export enum DataTopic {
+  Annotations = 'annotations',
+}
+
+/**
+ * In 8.2, this will become an interface
+ *
+ * @public
+ */
+export type DatasourceRef = string;
```

Comment:
In 8.2, this will become:  https://github.com/grafana/grafana/pull/33817/files#diff-2c9e457cb87628517f9e0606e5d774027ff002171070b0d561de1a178bf26e15R6

```
export interface DatasourceRef {
  type?: string;
  uid?: string;
}
```

I'm using string here just do decrease the number of files this PR will touch